src/app/static/src/tests/components/modelOutput/modelOutput.test.ts

```diff
@@ -122,6 +121,11 @@ function getStore(modelOutputState: Partial<ModelOutputState> = {}, partialGette
 declare let currentUser: string;
 currentUser = "guest";
 
+// will be easy to just remove this once feature switch enabled
+jest.mock("../../../app/featureSwitches",() => {
+    return {switches: {tableTab: true}}
+});
```

Comment:
Nice, I like that more than putting it in every test which is what I've done elsewhere!

Reply:
yhh thought since we are definitely adding this in, we just assume it is on in testing XD